ui/src/main.rs

```diff
@@ -270,7 +270,11 @@ fn main() {
     <body style="background-color:rgba(0,0,0,0);"><div id="main"></div></body>
     </html>"#
                     .to_string(),
-            ),
+            )
+            .with_file_drop_handler(|_w, drag_event| {
```

Comment:
Does this work across platform? I remember in the previous implementation of uplink that there was an issue with the file handler while working on linux. Not sure if thats still the case though